src/VirtualClient/VirtualClient.Actions.UnitTests/Redis/RedisServerExecutorTests.cs

```diff
@@ -38,7 +38,7 @@ public void SetupTests()
                 ["Port"] = 6379,
                 ["ServerInstances"] = 1,
                 ["ServerThreadCount"] = 4,
-                ["IsTLSEnabled"] = false
+                ["IsTLSEnabled"] = "no"
```

Comment:
Let's use the original boolean value vs. a string literal boolean. The property in your component can use the TryGetValue() method/technique for checking to see if the parameter is there and returning a default value if not.

Reply:
In the same profile we are using this value of IsTLSEnabled in the dependencies 
{
      "Type": "ExecuteCommand",
      "Parameters": {
        "Scenario": "CompileRedis",
        "SupportedPlatforms": "linux-x64,linux-arm64",
        "BUILD_TLS": "$.Parameters.IsTLSEnabled",
        "Command": "make BUILD_TLS={BUILD_TLS}",
        "WorkingDirectory": "{PackagePath:redis}"
      }
    }
BUILD_TLS takes "yes" or "no values to build Redis with TLS and without TLS respectively. To have this dependency as part of profile I was using IsEnabledTLS as string rather than as boolean.

If we could have ternary operator feature added in the profile expression evaluator, we can add ISEnabledTLS as true and just derive yes from this ISEnabledTLS value in the profile itself.